### PR TITLE
Feat/optim fid marker detection

### PR DESCRIPTION
**Pull request by yzhou-bcom**
Updated SolarPipelineTest_RelocalizationMarker and tested it with hololens images containing the fiducial marker

### CODE REVIEW

**Review by jim-bcom — DISMISSED** [review]

File: SolARPipeline_RelocalizationMarker/tests/SolARPipelineTest_RelocalizationMarker/markers.json [hunk 1]
```diff
@@ -1,85 +1,68 @@
 {
-	"nbTrackables": 4,	
+	"nbTrackables": 3,	
 	"trackable 0":{
 		"type": 1,
 		"data":{
-			"url": "FiducialMarkers.png",
+			"url": "FiducialMarkerA",
```
Comment (jim-bcom) [inline SolARPipeline_RelocalizationMarker/tests/SolARPipelineTest_RelocalizationMarker/markers.json:6]: Is this an actual URL, or just a name ?
FiducialMarkers.png still exists in the repo, is it still used ? If not, delete it ?

Comment (yzhou-bcom) [reply SolARPipeline_RelocalizationMarker/tests/SolARPipelineTest_RelocalizationMarker/markers.json:6]: FiducialMarkers.png is no longer used. It seems that in the latest version of markers.json file it has been deleted https://github.com/SolarFramework/Service-Relocalization/blob/develop/SolARService_RelocalizationMarkers/markers.json

Comment (jim-bcom) [reply SolARPipeline_RelocalizationMarker/tests/SolARPipelineTest_RelocalizationMarker/markers.json:6]: That's in Service-Relocalization, not Sample-Relocalization, and it still referenced in:
https://github.com/SolarFramework/Service-Relocalization/blob/develop/SolARService_RelocalizationMarkers/markers-2.json

Comment (yzhou-bcom) [reply SolARPipeline_RelocalizationMarker/tests/SolARPipelineTest_RelocalizationMarker/markers.json:6]: just asked Christophe, this file is no longer used in service reloc

**Pull request merged**